Formula/octave.rb

```diff
@@ -85,7 +93,8 @@ def install
               '""'
 
     # SUNDIALS 6.4.0 and later needs C++14 for C++ based features
-    ENV.append "CXXFLAGS", "-std=c++14"
+    # Configure to use gnu++14 instead of c++14 as octave uses GNU extensions
+    ENV.append "CXXFLAGS", "-std=gnu++14"
```

Comment:
Please append to `CXX` instead of `CXXFLAGS`. With the current flags, the tests during configure that are using the preprocessor (not the compiler) would not know about this flag. And they might give inconsistent results.

Reply:
👍